ccp/src/errors.rs

```diff
@@ -40,6 +40,9 @@ pub enum CCProverError {
 
     #[error(transparent)]
     IOError(#[from] tokio::io::Error),
+
+    #[error("utility thread closed channel")]
```

Comment:
The comment [here](https://github.com/fluencelabs/capacity-commitment-prover/pull/127/files#diff-f9b9eeefbbfbeea42d236f5ab6677f16f0b54a83479eb853ae3af09b1d46c053R338) tells the full queue causes the error. I think it is worth to sync the comment and the error msg.

Reply:
Indeed. Refactored utility errors.